### PR TITLE
Document the process for packaging a standalone .dacpac file

### PR DESCRIPTION
**Pull request by jeffrosenberg**
Per discussion on #67, this adds documentation to help users create a NuGet package from an existing .dacpac file. No code changes, documentation only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,7 +204,42 @@ Additionally you'll need to set the `PackageProjectUrl` property inside of the `
...
+  </metadata>
+  <files>
+    <file src="fileName.dacpac" target="tools/" />
+  </files>
+</package>
+```
+
+To create the package, run: 
+```
+nuget package fileName.nuspec
```
Comment (ErikEJ) [inline README.md:232]: nuget pack !

**Review by ErikEJ — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jeffrosenberg) [reply README.md]: Oops, sorry about that! 😬 Guess my fingers just completed the word and I didn't even notice! Fixed.

**Comment by jmezach** [comment]
Perhaps we should add a note about getting NuGet to work on Linux/Mac?

**Comment by jeffrosenberg** [comment]
@jmezach I'm not sure a .csproj makes sense for this particular use -- this is just a quick shim to get a standalone .dacpac into your local repo. I'll look at adding a note about installing NuGet locally once I'm done with work for the day.

**Comment by jeffrosenberg** [comment]
@jmezach I added a short note about installing the NuGet CLI tools, let me know if you were thinking of something more detailed than this.

**Review by jmezach — APPROVED** [review]

**Comment by jmezach** [comment]
@jeffrosenberg LGTM.

@ErikEJ Are you still waiting for more changes?

**Review by ErikEJ — APPROVED** [review]

**Pull request merged**